maps/CEVEris/_CEV_Eris.dmm

```diff
@@ -11662,11 +11665,8 @@
 /turf/simulated/floor/plating/under,
 /area/eris/maintenance/section3deck5starboard)
 "aCv" = (
-/obj/structure/disposalpipe/segment{
-	dir = 8;
-	icon_state = "pipe-c"
-	},
-/turf/simulated/floor/plating/under,
+/obj/structure/disposalpipe/segment,
+/turf/space,
```

Comment:
Not sure this is intented to add a space tile were an underplating was

Reply:
fixed